pkg/config/flags.go

```diff
@@ -0,0 +1,27 @@
+package config
+
+import (
+	"flag"
+	"strings"
+)
+
+type ServiceFlags []string
+
+func (s *ServiceFlags) String() string {
+	return strings.Join(*s, ",")
+}
+
+func (s *ServiceFlags) Set(value string) error {
+	*s = append(*s, value)
+	return nil
+}
+
+var (
+	ServicePaths ServiceFlags
+	ConfigPath   string
+)
+
+func init() {
```

Comment:
/blocker
Can we please put a direct "registerFlags" call in main?

Reply:
I believe all this stuff is gone. or should be gone if we didnt clean up properly. there is no use of flags anymore.